src/HorizontalTimeline/HorizontalTimeline.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Text from '../Text';
+import styles from './HorizontalTimeline.st.css';
+import iconsStyles from './HorizontalTimelineIcons.st.css';
+import StatusCompleteFilled from 'wix-ui-icons-common/StatusCompleteFilled';
+import StatusAlertFilled from 'wix-ui-icons-common/StatusAlertFilled';
+import Box from '../Box';
+
+class HorizontalTimeline extends React.PureComponent {
+  render() {
+    const { items, dataHook, className } = this.props;
+
+    return (
+      <div {...styles('root', {}, { className })} data-hook={dataHook}>
+        {items.map(({ label, width, skin, icon }, i) => {
+          width = width || 'auto';
+          skin = skin || 'light';
+          icon = icon || <HorizontalTimeline.DefaultIcon />;
+
+          const isNextItemDark = items[i + 1] && items[i + 1].skin === 'dark';
+
+          return (
+            <div className={styles.column} key={i} style={{ width: width }}>
+              <div className={styles.item}>
+                <div className={styles.topRow}>
+                  <div {...styles('line', { skin })} />
+                  <div className={styles.iconWrapper}>{icon}</div>
+                  <div
+                    {...styles('line', {
+                      skin: isNextItemDark ? 'dark' : 'light',
+                    })}
+                  />
+                </div>
+
+                <Box className={styles.label}>
+                  <Text size="tiny" secondary={skin === 'light'} ellipsis>
```

Comment:
Same comment as above, this text is very specific for the 2 skins.
You can create a `textPropsBySkinMap` variable (outside the component) that stores text props per skin.

For example:
```
const skinTextPropsMap = {
  dark: {},
  light: { secondary: true },
  destructive: { skin: 'destructive' },
  success: { skin: 'success' },
};
```
```suggestion
                  <Text size="tiny" ellipsis {...skinTextPropsMap[skin]} >
```

Reply:
Done